Resources/Views/AdminPanel/Layout/Partials/alerts.leaf

```diff
@@ -1,7 +1,7 @@
 <div class="alerts">
 #flash() {
     #for(flash in all) {
-        #bs:alert(flash.bootstrapClass) {
```

Comment:
Ah now I understand.. I didn't push the Flash updates. Do you mind changing this to `bootstrapClass`? Then I'll push the Flash update.